beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/api/BeekeeperApiIntegrationTest.java

```diff
@@ -0,0 +1,163 @@
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.springframework.http.HttpStatus.OK;
+
+import static com.expediagroup.beekeeper.api.response.MetadataResponseConverter.convertToHousekeepingMetadataResponse;
+
+import java.io.IOException;
+import java.net.http.HttpResponse;
+import java.sql.SQLException;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.data.domain.Page;
+import org.springframework.util.SocketUtils;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.ValueInstantiationException;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+
+import com.expediagroup.beekeeper.api.BeekeeperApiApplication;
+import com.expediagroup.beekeeper.api.response.HousekeepingMetadataResponse;
+import com.expediagroup.beekeeper.core.model.HousekeepingMetadata;
+import com.expediagroup.beekeeper.core.model.HousekeepingStatus;
+import com.expediagroup.beekeeper.core.model.LifecycleEventType;
+import com.expediagroup.beekeeper.integration.BeekeeperIntegrationTestBase;
+import com.expediagroup.beekeeper.integration.utils.BeekeeperApiTestClient;
+import com.expediagroup.beekeeper.integration.utils.RestResponsePage;
+
+public class BeekeeperApiIntegrationTest extends BeekeeperIntegrationTestBase {
+
+  @Bean
+  @Primary
+  public ObjectMapper geObjMapper() {
+    return new ObjectMapper()
+        .registerModule(new ParameterNamesModule())
+        .registerModule(new Jdk8Module())
+        .registerModule(new JavaTimeModule());
+  }
+
+  private static final Logger log = LoggerFactory.getLogger(BeekeeperApiIntegrationTest.class);
+
+  // APP CONTEXT AND TEST CLIENT
+  protected static ConfigurableApplicationContext context;
+  protected BeekeeperApiTestClient testClient;
+  
+  protected final ObjectMapper mapper = geObjMapper();
+
+  @BeforeEach
+  public void beforeEach() {
+    mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+
+    int port = SocketUtils.findAvailableTcpPort();
+    String[] args = new String[] {
+        "--server.port=" + port};
```

Comment:
Formatting looks weird. Either put that brace on a new line or everything on one line